chpl/chpl-service/src/main/java/gov/healthit/chpl/manager/impl/SchedulerManagerImpl.java

```diff
@@ -238,6 +238,22 @@ public void retireAcb(final String acb) throws SchedulerException, ValidationExc
         }
     }
 
+    @Override
+    @PreAuthorize("hasAnyRole('ROLE_ADMIN', 'ROLE_ONC', 'ROLE_ACB')")
```

Comment:
Should this use the new security model?

Reply:
It certainly could be, but I wasn't prepared to make an entire transition of the manager. There's another discussion we need to have regarding Quartz jobs and how those are handled by security too, and I'm hoping both of those can be resolved at the same time